po/tr.po

```diff
@@ -262,15 +262,15 @@ msgstr "Satın Alma veya Fon…"
 
 #: src/Views/ErrorView.vala:28
 msgid "Whoops"
-msgstr "Hoppala vede Cupala :D"
+msgstr "Whoops"
 
 #: src/Views/ErrorView.vala:31
 msgid "Could not display the page."
 msgstr "Sayfa görüntülenemedi."
 
 #: src/Views/WelcomeView.vala:31
 msgid "The always-incognito web browser"
-msgstr "Her zaman gizli web tarayıcısı"
+msgstr "Gizli web tarayıcınız"
```

Comment:
This seems to be a less direct translation, though I'm not sure if it makes more sense or not.

Reply:
More understandable such sentence
Same as narration